cmd/buildah/images_test.go

```diff
@@ -74,14 +74,14 @@ func TestSizeFormatting(t *testing.T) {
 		t.Errorf("Error formatting size: expected '%s' got '%s'", "0 B", size)
 	}
 
-	size = formattedSize(1024)
+	size = formattedSize(1000)
 	if size != "1 KB" {
 		t.Errorf("Error formatting size: expected '%s' got '%s'", "1 KB", size)
 	}
 
-	size = formattedSize(1024 * 1024 * 1024 * 1024 * 1024)
-	if size != "1024 TB" {
-		t.Errorf("Error formatting size: expected '%s' got '%s'", "1024 TB", size)
+	size = formattedSize(1000 * 1000 * 1000 * 1000)
```

Comment:
Did you lose a * 1000 in here? The original multiplies 1000 five times, this only does 4

Reply:
Yes that was intentional.  I saw no reason to have 1000 TB.

Reply:
Should you change the expected below to be 1TB instead of 1000TB then?

Reply:
Fixed